rs/execution_environment/benches/wasm_instructions/helper.rs

```diff
@@ -24,40 +24,54 @@ pub fn first_or_all<'a>(all: &'a [&'a str]) -> &'a [&'a str] {
 ///
 /// The confirmation benchmark is to make sure there is no compiler optimization
 /// for the repeated lines of code.
-pub fn benchmark_with_confirmation(name: &str, code: &str) -> Vec<common::Benchmark> {
+pub fn benchmark_with_confirmation(
+    name: &str,
+    code: &str,
+    wasm64_enabled: bool,
+) -> Vec<common::Benchmark> {
     let i = DEFAULT_LOOP_ITERATIONS;
     let r = DEFAULT_REPEAT_TIMES;
     let c = CONFIRMATION_REPEAT_TIMES;
     vec![
-        benchmark(name, i, r, code),
-        benchmark(&format!("{name}/confirmation"), i, c, code),
+        benchmark(name, i, r, code, wasm64_enabled),
+        benchmark(&format!("{name}/confirmation"), i, c, code, wasm64_enabled),
     ]
 }
 
 /// Creates a benchmark with its confirmation for the specified `code` snippet.
 ///
 /// The confirmation benchmark is to make sure there is no compiler optimization
 /// for the loop.
-pub fn benchmark_with_loop_confirmation(name: &str, code: &str) -> Vec<common::Benchmark> {
+pub fn benchmark_with_loop_confirmation(
+    name: &str,
+    code: &str,
+    wasm64_enabled: bool,
+) -> Vec<common::Benchmark> {
     let i = DEFAULT_LOOP_ITERATIONS;
     let c = CONFIRMATION_LOOP_ITERATIONS;
     let r = DEFAULT_REPEAT_TIMES;
     vec![
-        benchmark(name, i, r, code),
-        benchmark(&format!("{name}/confirmation"), c, r, code),
+        benchmark(name, i, r, code, wasm64_enabled),
+        benchmark(&format!("{name}/confirmation"), c, r, code, wasm64_enabled),
     ]
 }
 
 /// Creates a benchmark with a code block repeated specified number of times in a loop.
-pub fn benchmark(name: &str, i: usize, r: usize, repeat_code: &str) -> common::Benchmark {
+pub fn benchmark(
+    name: &str,
+    i: usize,
+    r: usize,
+    repeat_code: &str,
+    wasm64_enabled: bool,
+) -> common::Benchmark {
```

Comment:
I think if we just return Wasm32 + Wasm64 benchmarks here, it would minimize the changes across files. Alternatively, we can call this function twice from `benchmark_with_confirmation` and `benchmark_with_loop_confirmation`, in wasm32 and wasm64 modes.